requirements/dev.txt

```diff
@@ -33,6 +33,14 @@ transifex-client==0.12.4
 # Dependency of transifex-client
 urllib3==1.20
 
+# Needed for documentation_crawler
+pathlib2==2.2.1
+#Needed for pathlib2==2.2.1
+scandir==1.5
+
```

Comment:
But does that mean that we don't need this listed here, since these are only recursive dependencies?